index.html

```diff
@@ -1,176 +1,780 @@
+            <div class="icon"></div>
+            <h4>Find recipes</h4>
+            <p>Find recipes suited to all levels, from student to master chef, and check their environmental impact. Coming soon... make your own recipes! </p>
+            <!--
+            <div class="text-button">
+              <a href="#">Read More <i class="fa fa-arrow-right"></i></a>
+            </div>
+          -->
+          </div>
+        </div>
+        <div class="col-lg-3">
+          <div class="service-item third-service">
+            <div class="icon"></div>
+            <h4>Diet calendar</h4>
+            <p>Plan and log your meals. We check for possible nutrient deficiencies and provide stats on your nutrition and environmental impact.</p>
+            <!--
+            <div class="text-button">
+              <a href="#">Read More <i class="fa fa-arrow-right"></i></a>
+            </div>
+          -->
+          </div>
+        </div>
+        
+        <div class="col-lg-3">
+          <div class="service-item fourth-service">
+            <div class="icon"></div>
+            <h4>Count calories, count CO2 too!</h4>
+            <p>Set targets for climate impact and nutrition. Find alternatives that can make a difference and smash down your goals!</p>
+             <!--
+            <div class="text-button">
+              <a href="#">Read More <i class="fa fa-arrow-right"></i></a>
+            </div>
+          -->
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
 
-                // Get all elements with class="tablinks" and remove the class "active"
-                tablinks = document.getElementsByClassName("tablinks");
-                for (i = 0; i < tablinks.length; i++) {
-                    tablinks[i].className = tablinks[i].className.replace(" active", "");
-                }
+  <div id="about" class="about-us section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-6 align-self-center">
+          <div class="section-heading">
+            <h4>About us</h4>
+            <img src="assets/images/heading-line-dec.png" alt="">
+            <p>At Quanteco we are working towards a future where we can track our entire carbon footprint online, and live our lives mindful of the effects of our lifestyle. We believe that with an understanding of our true impact as individuals, we can harness our competitive and social potential to solve the climate crisis.
+            </br>
+              Using the science of life cycle analysis, we strip away the layers of industry between farm and plate, so that normal people can make decisions that improve the agricultural techniques of the world.</p>
+      
+            </p>
+          <div class="row">
+            <div class="col-lg-6">
+              <div class="box-item">
+                <h5> Robert - CEO </h5>
+               <p><img class="teamimage" src="assets/images/qe/robert.png" style="height: 267px; width: 175px;" alt="Robert"> <br></p>
```

Comment:
We need more professional, consistent, and equally sized images :)

Reply:
Let's take them then!